mne/viz/evoked.py

```diff
@@ -126,6 +127,35 @@ def _rgb(x, y, z):
     return np.asarray([x, y, z]).T
 
 
+def _plot_legend(pos=None, colors=None, axis=None, outlines='head'):
+    """Helper function to plot color/channel legends for butterfly plots
+    with spatial colors"""
+    from mpl_toolkits.axes_grid.inset_locator import inset_axes
+    ax = inset_axes(axis, width=0.5, height=0.5, loc=3)
```

Comment:
And here I get ``AttributeError: 'AnchoredSizeLocator' object has no attribute '_child'``